README.md

```diff
@@ -4,12 +4,29 @@ WORK IN PROGRESS. Right now this connects to the MAAP STAC - providing a module
 
 Much of this project is inspired from [leafmap](https://leafmap.org/)
 
-![Jupyter Lab ScreenShot](jlab-screenshot.png)
+![Jupyter Lab ScreenShot](/public/images/jlab-screenshot.png)
 
 ## Additional requirements
 
 * jupyter lab, node>=12.0.0
 
+## Getting started locally
+This project will run locally as a dev container which will need to be installed. If you are using VSCode, add the `Dev Containers` extension and then also install the `Remote Development` extensions. Conda is used to manage our base environment packages and then we will use pip to manage our libraries.
+
+Once these have been installed, follow these steps below:
+1. To start the dev container, click on the `Remote Development` extensions icon in the bottom left corner, and then from the options list choose `Reopen in Container`
+2. Making sure you are at the root of `stac_ipyleaflet`, In the terminal in VSCode, run `conda activate base`. Your terminal should look something like this... with the red text referencing your branch name
```

Comment:
The color of the text may be different - the screenshot & desc may be enough here.